Source/ORTS.Settings/UserSettings.cs

```diff
@@ -251,6 +251,8 @@ public enum DirectXFeature
         public bool OverrideNonElectrifiedRoutes { get; set; }
         [Default(true)]
         public bool HotStart { get; set; }
+        [Default(false)]
+        public bool NoDieselEngineStart { get; set; }
```

Comment:
Tying the power mode to a global setting like this seems too inflexible; surely this should be a property of the selected consist and/or the selected activity. For example, imagine an activity that starts your locomotive in diesel mode in a non-electrified portion of track, with the expectation that you will change over to electric mode upon reaching the electrified section. The activity designer would want to ensure that the player's locomotive starts up in the correct mode.

You also have to consider other kinds of locomotives that can't be captured with a simple diesel on/diesel off toggle, such as dual overhead/third-rail units.

In the interest of getting this PR through, I would support keeping this setting for now, but I would like to move it to the Experimental tab. In the long-term, the goal should be to make it easy for the player to select which mode to start up in in explorer mode, and for activity and timetable designers to be able to select which modes they want their trains to start up in, thus allowing us to remove it.

Reply:
Ok to put it in the Experimental tab for the moment.
A parameter should be added to the timetables later.